src/services/session.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const protocolCallbacks = require('@services/protocolCallbacks/')
+const sessionAttendanceQueries = require('@database/storage/sessionAttendance/queries')
+const userQueries = require('@database/storage/user/queries')
+const bapQueries = require('@database/storage/bap/queries')
+const crypto = require('crypto')
+
+exports.session = async (requestBody) => {
+	try {
+		const sessionId = requestBody.sessionId
+		const sessionAttendance = await sessionAttendanceQueries.findBySessionId(sessionId)
```

Comment:
"sessionAttendances" would be more apt here since you are fetching multiple sessionAttendances with common sessionId.